sample_app/config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  
+  root 'static_pages#home'
+
+  get 'static_pages/home'
+  get 'static_pages/help'
+  get 'static_pages/about'
+  get 'static_pages/contact'
```

Comment:
I'm pretty positive the tutorial walks me through how to change that in later chapters.  Let's leave it for now.